lizmap/modules/lizmap/controllers/features.classic.php

```diff
@@ -170,13 +170,23 @@ public function displayExpression()
 
         $expressions = array_merge($expressions, $additionalFields);
 
+        $sortingField = trim($this->param('sorting_field', ''));
+
+        $sortingOrder = trim($this->param('sorting_order', 'asc'));
+
+        // Limit
+        $limit = trim($this->param('limit', ''));
```

Comment:
```suggestion
        $limit = trim($this->param('limit', 1000));
```
As default virtualFields $limit value.